XamarinCommunityToolkit.UnitTests/ConvertersNamespace_Tests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using System.Reflection;
+using Xamarin.CommunityToolkit.Behaviors;
+using Xamarin.Forms;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests
+{
+	public class Namespace_Tests
+	{
+		[Fact]
+		public void MakeSureConvertersAreInTheRightNamespace()
+		{
+			var allTheTypes = Assembly.GetAssembly(typeof(BaseBehavior)).GetTypes();
+			foreach (var type in allTheTypes.Where(t => t.Name.EndsWith("Converter") && t.GetInterface("IValueConverter") != null))
+			{
+				Assert.True(type.Namespace.Equals("Xamarin.CommunityToolkit.Converters"), $"{type.FullName} not in Xamarin.CommunityToolkit.Converters namespace");
+			}
+		}
+
+		[Fact]
+		public void MakeSureEffectsAreInTheRightNamespace()
+		{
+			var allTheTypes = Assembly.GetAssembly(typeof(BaseBehavior)).GetTypes();
+			foreach (var type in allTheTypes.Where(t => t.Name.EndsWith("Effect") && t.IsClass && t.IsSealed && t.IsAbstract))
+			{
+				Assert.True(type.Namespace.Equals("Xamarin.CommunityToolkit.Effects"), $"{type.FullName} not in Xamarin.CommunityToolkit.Effects namespace");
+			}
+		}
+
+		[Fact]
+		public void MakeSureMarkupExtensionsAreInTheRightNamespace()
+		{
+			var allTheTypes = Assembly.GetAssembly(typeof(BaseBehavior)).GetTypes();
+			foreach (var type in allTheTypes.Where(t => t.Name.EndsWith("Extension") && t.GetInterface("IMarkupExtension") != null))
+			{
+				Assert.True(type.Namespace.Equals("Xamarin.CommunityToolkit.Extensions"), $"{type.FullName} not in Xamarin.CommunityToolkit.Extensions namespace");
+			}
+		}
+
+		[Fact]
+		public void MakeSureBehaviorsAreInTheRightNamespace()
+		{
+			var allTheTypes = Assembly.GetAssembly(typeof(BaseBehavior)).GetTypes();
+			foreach (var type in allTheTypes.Where(t => t.Name.EndsWith("Behavior") && t.IsSubclassOf(typeof(BaseBehavior))))
+			{
+				Assert.True(type.Namespace.Equals("Xamarin.CommunityToolkit.Behaviors"), $"{type.FullName} not in Xamarin.CommunityToolkit.Behaviors namespace");
+			}
+		}
+
+		[Fact]
+		public void MakeSureViewsAreInTheRightNamespace()
+		{
+			var allTheTypes = Assembly.GetAssembly(typeof(BaseBehavior)).GetTypes();
+			foreach (var type in allTheTypes.Where(t => t.Name.EndsWith("View") && t.IsSubclassOf(typeof(View))))
```

Comment:
Mmm, e.g. Expander won't have "View" postfix.
Will we add it? Or Should we workaround it somehow?

Reply:
I've removed the check to end in view and just see if it inherits from it